lib/screens/task_list_screen.dart

```diff
@@ -59,4 +83,24 @@ class _TaskListScreenState extends State<TaskListScreen> {
       ],
     );
   }
+
+  Widget _buildTaskContainer(Task task, int index) {
+    return Container(
+      color: index.isEven ? Colors.white : Colors.white30,
+      child: ListTile(
+        leading: task.completed
+            ? Icon(Icons.check_circle, color: Colors.green)
+            : Icon(Icons.circle_outlined),
```

Comment:
Circle outlined is indeed not available in the current channel. (Although it's listed in the flutter docs.)

See also the issues:
* https://github.com/flutter/flutter/issues/27956, 
* https://github.com/flutter/flutter/pull/61778
* https://github.com/flutter/flutter/issues/82349
* etc.

Two possible solutions would be:
* using the package material_design_icons_flutter
* switching to a different channel

Regarding the latter option they recommend to use the stable channel (which we are in currently). That said, they also state using the beta channel should be fine because it serves just as an additional maturing phase for a stable release without any extra testing involved. See https://github.com/flutter/flutter/wiki/Flutter-build-release-channels for more.

I just tried it out with the beta channel and it seems to work. The question remains, should we switch into the beta channel or better use the package? (Theoretically we could also just include the material icon font ourselves as an asset and used it from there.) What's your opinion on that matter?

Reply:
Thanks for your input. For the moment, I changed the Icon to a `radio_button_off` icon and added a semanticLabel to make the meaning of the icon also clear for screen readers. I also have the rational for this change in the commit message, so git blame will explain the change. I just realized, I could have added a comment in the code as well, which I am happy to add with my next commit.